src/web3/getNetworkName.js

```diff
@@ -16,9 +16,11 @@ export async function getNetworkName () {
     networkName = network.name
   } else {
     if (!process.env.REACT_APP_DEFAULT_NETWORK_NAME) {
-      throw new Error('You must define the enviroment variable REACT_APP_DEFAULT_NETWORK_NAME')
+      networkName = 'homestead'
```

Comment:
`homestead` is a rather weird word to throw in here. That's an ethers thing?

Reply:
Yeah it's an Ethers.js thing.  However, "homestead" is actually the official name of the first network.